test/bus_test.dart

```diff
@@ -13,17 +13,46 @@ import 'package:test/test.dart';
 import 'package:rohd/src/utilities/simcompare.dart';
 
 class BusTestModule extends Module {
+  // --- Getters ---
   Logic get aBar => output('a_bar');
   Logic get aAndB => output('a_and_b');
-  Logic get aShrunk => output('a_shrunk');
-  Logic get aRSliced => output('a_rsliced');
-  Logic get aReversed => output('a_reversed');
-  Logic get aRange => output('a_range');
   Logic get aBJoined => output('a_b_joined');
   Logic get a1 => output('a1');
   Logic get aPlusB => output('a_plus_b');
 
+  // Getters for Logic subset test variables
+  Logic get aReversed => output('a_reversed');
+  // --- Getters for Slicing
+  Logic get aShrunk1 => output('a_shrunk1');
+  Logic get aShrunk2 => output('a_shrunk2');
+  Logic get aShrunk3 => output('a_shrunk3');
+  Logic get aNegShrunk1 => output('a_neg_shrunk1');
+  Logic get aNegShrunk2 => output('a_neg_shrunk2');
+  Logic get aNegShrunk3 => output('a_neg_shrunk3');
+  // --- Getters for Reverse Slicing
+  Logic get aRSliced1 => output('a_rsliced1');
+  Logic get aRSliced2 => output('a_rsliced2');
+  Logic get aRSliced3 => output('a_rsliced3');
+  Logic get aRNegativeSliced1 => output('a_r_neg_sliced1');
+  Logic get aRNegativeSliced2 => output('a_r_neg_sliced2');
+  Logic get aRNegativeSliced3 => output('a_r_neg_sliced3');
+  // --- Getters for getRange
+  Logic get aRange1 => output('a_range1');
+  Logic get aRange2 => output('a_range2');
+  Logic get aRange3 => output('a_range3');
+  Logic get aNegativeRange1 => output('a_neg_range1');
+  Logic get aNegativeRange2 => output('a_neg_range2');
+  Logic get aNegativeRange3 => output('a_neg_range3');
+  // --- Getters for operator[]
+  Logic get aOperatorIndexing1 => output('a_operator_indexing1');
```

Comment:
do you have tests actually using these outputs somewhere?

Reply:
yes, I do. will be merging your latest changes and verify these change with this PR

Reply:
these variables are used for different corner testcases

Reply:
that are within the test cases for getRange() in bus_test.dart